src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -38,78 +39,42 @@
 	private static final Pattern intPattern = Pattern.compile("([\\+-]?\\d+)([eE][\\+-]?\\d+)?");
 //	private static final Pattern floatPattern = Pattern.compile("([\\+-]?\\d(\\.\\d*)?|\\.\\d+)([eE][\\+-]?(\\d(\\.\\d*)?|\\.\\d+))?");
 
-	public static final int BITRATE_DEFAULT = 8000;
-	public static final int NUMBEROFCHANNELS_DEFAULT = 2;
-	public static final int BITSPERSAMPLE_DEFAULT = 16;
-	public static final int AUDIODELAY_DEFAULT = 0;
-	public static final int SAMPLEFREQUENCY_DEFAULT = 48000;
-
-	private int bitRate;
 	private int numberOfChannels;
-	private int bitsperSample;
 	private int audioDelay;
 	private int sampleFrequency;
 
 	public int getAttribute(AudioAttribute attribute) {
 		switch (attribute) {
-			case BITRATE:
```

Comment:
@Sami32 I'm wishing to, but I'm not intending to - because of backwards compatibility. Doing things like that will just break stuff. 

If it were up to me alone, I would make a "clean cut" at some major version and remove all deprecated code with a new major version. That way plugins would have to be updated for let's say UMS 8.x.x, but would work for any previous version without update. It's not up to me though, so I'm not going to push for it.